libs/core/src/lib/components/wrapper-component.ts

```diff
@@ -248,8 +247,10 @@ export abstract class ReactWrapperComponent<TProps extends {}> implements AfterV
       additionalProps?: ReactContentProps;
     }
   ): (props?: TProps, defaultRender?: JsxRenderFunc<TProps>) => JSX.Element | null {
-    if (renderInputValue instanceof RenderOverride) {
-      return renderInputValue.renderer;
+    if ((typeof renderInputValue === 'object') && ('getProps' in renderInputValue)) {
```

Comment:
Please use a user-defined type guard here, as mentioned in the other comment. Making one will also make sure you can re-use the logic of it in multiple places.